### PR TITLE
results in main editor fixes #56

### LINKED ISSUE #56: Show results in editor left panel as a source tab and disable folding in right plugin tab.
Currently we allow folded code in intellij plugin right tab, which on being expanded leads to occupies to whole space and makes it very difficult to navigate and view code. After this user will be able to view suggestion as highlighted text in the same editor as he is programming but as a new tab.

This task includes highlighting results.

### PR DESCRIPTION
**Pull request by prudhvije**

### CODE REVIEW

**Comment by ScrapCodes** [comment]
make editors read only

**Comment by ScrapCodes** [comment]
Can you also add opening of editor in right click menu.

**Comment by prudhvije** [comment]
Right clicking itself opening file in editor. Cause it's also selection. Shall we further give one more item in right click menu?

File: plugins/idea/betterdocsidea/src/main/java/com/imaginea/betterdocs/EditorDocOps.java [hunk 1]
```diff
@@ -233,4 +253,90 @@ private void getCompletePackage(final PsiPackage completePackage,
...
+        String filePath = tempDir + "/" + fileName;
+        File file = new File(filePath);
+        if (!file.exists()) {
+            try {
+                file.createNewFile();
+            } catch (IOException e1) {
+                e1.printStackTrace();
+            }
+        } else {
+            VirtualFile virtualFile = LocalFileSystem.getInstance().findFileByPath(filePath);
```
Comment (prudhvije) [inline plugins/idea/betterdocsidea/src/main/java/com/imaginea/betterdocs/EditorDocOps.java]: Set Writable to true else permission denied exception. Set writable to false once writing is done.

**Comment by prudhvije** [comment]
image

**Comment by prudhvije** [comment]
image

**Comment by prudhvije** [comment]
Main Pane:
Jtree

Selection -> show code in window editor
Right Click -> Option to open in new tab

Code Pane:
Click to open in new tab.

image

**Pull request closed without merging**